snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -386,7 +387,10 @@ def ignore(directory, files):
         shutil.copytree(self.code.sourcedir, self.code.build_basedir,
                         symlinks=True, ignore=ignore)
 
+        script_runner = ScriptRunner(builddir=self.code.build_basedir)
+
         self.code.build()
+        script_runner.install(scriptlet=self._part_properties.get('install'))
```

Comment:
it's not clear why you initialize script_runner, then do something else, and then use it.
If there's a reason for that, can you explain it in a comment?